tutorials/dataframe/df023_aggregate.C

```diff
@@ -10,7 +10,7 @@
 /// \macro_output
 ///
 /// \date July 2018
-/// \author Enrico Guiraud, Danilo Piparo CERN, Massimo Tumolo Politecnico di Torino
+/// \authors Enrico Guiraud, Danilo Piparo, Massimo Tumolo
```

Comment:
why removing the affiliations, here and below?

Reply:
Because the affiliation was nowhere else except for a few tutorials in dataframe.. that's just to be consistent.

Reply:
But if you prefer to have it I can do the opposite and add the affiliation where it was missing.

Reply:
i think the idea was to show that these tutorials were contributed by developers from different orgs, not just the core ROOT devs. Personally I don't mind the inconsistency, but if you really want to remove them that's ok.

Reply:
You have a good point. I will put them back.

Reply:
Done, they are back.